metadata/classes/ContactListController.cls

```diff
@@ -4,4 +4,10 @@ public class ContactListController {
         return [SELECT Id, Name, Email, Title, Phone, Profile_Image__c FROM Contact
                 ];        
     }
+
```

Comment:
If the table in this query contains more than 50'000 records a runtime exception will be thrown. I suggest to introduce a `WHERE` clause or cap the max number of records returned adding a `LIMIT` clause to the query.